python/py-setuptools_scm/Portfile

```diff
@@ -25,7 +25,9 @@ checksums           rmd160  bdddb3b45653392ac098004c2fd9ee166015cd63 \
                     size    45549
 
 if {${name} ne ${subport}} {
-    depends_lib-append  port:py${python.version}-setuptools
+    depends_build-append port:py${python.version}-setuptools \
+                         
+    depends_run-append   port:py${python.version}-toml
```

Comment:
It's listed as ```extra``` dependency so not strictly necessary for its functioning. If other ports depend on ```setuptools-scm[toml]``` you could just add a build dependency there on ```py-toml```. On the other hand, it's likely that ports will start using this more and then it could make sense to add it here as default. I am not sure what would be the best.

Reply:
We do use of for building setuptools as we store build logic in there pep 517👍